database/factories/HitFactory.php

```diff
@@ -0,0 +1,16 @@
+<?php
+
+namespace Database\Factories;
+
+use Illuminate\Database\Eloquent\Factories\Factory;
+
+class HitFactory extends Factory
+{
+    public function definition()
+    {
+        return [
+            'endpoint' => '/api/releases/8.0',
```

Comment:
This is probably silly, but could we make this a tiny bit more random by making the version number randomized?